cmd/bbolt/surgery_command_cobra.go

```diff
@@ -0,0 +1,71 @@
+package main
```

Comment:
I propose to rename this file as `cmd_surgery.go` or `command_surgery.go`

Reply:
Agreed. Actually I am thinking to migrate all existing commands to cobra style and refactor the naming in followup PRs.

Reply:
Changed to `command_surgery_cobra.go` for now. Will remove the suffix "_cobra" after we migrate all commands to cobra style.